orbstation/quirks/robot_voice.dm

```diff
@@ -0,0 +1,36 @@
+//Code largely copied from the "tongue tied" quirk. This one is much simpler, though.
+
+/datum/quirk/robot_voice
+	name = "Artificial Voice Box"
+	desc = "Your tongue and throat were damaged beyond repair in some past incident, and now you have a robotic voice box to replace them. \
+	Note that your robotic voice box cannot be removed."
+	icon = "robot"
+	value = 0 //the robot tongue is capable of speaking every language but does not actually grant them to you - so it doesn't do much
+
+/datum/quirk/robot_voice/add_unique()
+	var/mob/living/carbon/human/human_holder = quirk_holder
+	var/obj/item/organ/internal/tongue/old_tongue = human_holder.getorganslot(ORGAN_SLOT_TONGUE)
+	old_tongue.Remove(human_holder)
+	qdel(old_tongue)
```

Comment:
I know this was borrowed from other code, but is this step necessary?
It looks like you might just be able to do ` new_tongue.Insert(human_holder, drop_if_replaced = FALSE)` and it will delete the old one for you.

Reply:
This did work, thank you.